azerothcore/azerothcore-wotlk
# [Core/Pathfinding] Some improvements... hopefully...

# CHANGES PROPOSED:
- Reduce "Max Walkable Angle" to 60*
- Use pathfinding for Point movement.
- Reduce fleeing movement (Not In LoS) check to 100ms


## ISSUES ADDRESSED:
- Closes #2688
- Closes #2912 
- Closes #862 (Potentially)
- Closes #948 (Potentially)
- Closes #2124(Potentially) 

## TESTS PERFORMED:
Below.


## HOW TO TEST THE CHANGES:
As of the issues, example:

```
    .cheat god
    .go cr 90632 (inside BRD)
    Attack this mob but do not kill it, at around 20% he will flee and join the other side on the right and will pull all the mobs which is normally impossible and will wipe your group. This is kinda gamebreaking (and ugly to see).
    You can also jump below and they will all follow you.
```

## KNOWN ISSUES AND TODO LIST:
None that I know of...

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
